amplify/backend/backend-config.json

```diff
@@ -292,6 +292,67 @@
           ]
         }
       ]
+    },
+    "AddItem": {
+      "build": true,
+      "providerPlugin": "awscloudformation",
+      "service": "Lambda",
+      "dependsOn": [
+        {
+          "category": "storage",
+          "resourceName": "main",
+          "attributes": [
+            "Name",
+            "Arn",
+            "StreamArn"
```

Comment:
Remove `StreamArn`. None of the tables currently do not support DynamoDB Streams (You can look up what that is if you want).